app/controllers/assignments_controller.rb

```diff
@@ -129,8 +129,12 @@ def set_assignment
     @assignment = @organization.assignments.includes(:assignment_invitation).find_by!(slug: params[:id])
   end
 
+  def set_list_type
+    @list_type = @organization.roster ? :roster_entries : :assignment_repos
```

Comment:
I've created a `@list_type` instance variable set in a `before_action` to help make the view logic a little more readable, whereas previously were only relying `if @organization.roster` scattered everywhere for the same functionality